test/tests/functional/pbs_multi_sched.py

```diff
@@ -2084,3 +2083,16 @@ def test_multi_sched_priority_sockets(self):
         self.server.manager(MGR_CMD_SET, SCHED,
                             {'scheduling': 'True'}, id='sc2')
         self.server.log_match("processing priority socket", starttime=t)
+
+    def tearDown(self):
+        if TestMultipleSchedulers.rlimit_flag == 1:
```

Comment:
instead of writing a tearDown() which is effective only for one test (at this time) and we need to use class variables to do that, how about handling this in the test case itself, where regardless of the test result, we revert the limits, maybe by using PtlExpectError.

Reply:
Hi @prakashcv13, If I am not wrong, PtlExpectError is for catching the expect statements, after setting the rlimit, the test can fail at any point. I agree with you regarding having teardown is kind of trivial for now. I am just thinking to add try...except...finally for rest of the test case after the limit is set. How does that sound?

Reply:
you are right, I just assumed that the test case would fail at the except(). Your idea sounds good to me.

Reply:
@prakashcv13 I have addressed your suggestions. Please look at the latest commit.